layouts/partials/helpers/image.html

```diff
@@ -1,24 +1,43 @@
 {{/* Global resource fallback */}}
 {{- $image := .asset.image -}}
+
+{{/* Resource is used when file is not imported from static directory, in which
```

Comment:
Not sure what you want to say here. Might just be me, but maybe could be described in more detail.